docs/components/HeaderFileIntro.mdx

```diff
@@ -12,18 +12,16 @@ export default {
 }
 ```
 
-Into:
+Into that:
 
 ```js
-// /pages/some-page/+config.h.js
-
-const Page = 'import:./some-file';
-const onBeforeRender = 'import:./some-other-file';
+const Page = 'import:./some-file:default';
+const onBeforeRender = 'import:./some-other-file:default';
 
 export default {
   Page,
   onBeforeRender
 }
 ```
 
-> `.h.js` files are only meant for config files (i.e. any `.h.js` file should be a `+config.h.js` file).
+> This transformation is only for config files. (Any `.h.js` file is a `+config.h.js` file.)
```

Comment:
Does it mean "only `+config.h.js` is supported"? or does it mean "`anything.h.js` will be treated as `+config.h.js`"?

Reply:
Could I create a `+Page.h.js`  to feed the Page config and have transformed imports in there, even if this makes no sense?

Reply:
2e58a6c33bd22ee07bd84231c1945b7a6dcc43cf &mdash; hopefully it's clearer now.